src/arrayMethodReduce.js

```diff
@@ -5,7 +5,21 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let acc = initialValue;
```

Comment:
I don't understand this name of variable(
Code is nice)
Good job)

Reply:
Thanks!
"acc" - it's short from "accumulator" - reduce(accumulator, currentValue),
that's the same what Misha writes "prev"